src/locales/default.json

```diff
@@ -904,7 +904,7 @@
     "stakingIncentives": "Staking incentives",
     "totalShareTooltip": "Your current total share of LP tokens in the pool",
     "restakeGauge": "Restake to receive future liquidity mining incentives",
-    "restakeGaugeDescription": "You’re staking against a pool gauge which is being deactivated on 21 September 2022. Restake to the new gauge for this pool to receive future liquidity mining incentives.",
+    "restakeGaugeDescription": "You are staking in a gauge which has been deprecated as of May 25th, 2023. Restake to the new gauge for this pool to receive future liquidity mining incentives.",
```

Comment:
I think we should maybe remove reference to dates here. Lets hold off merging until we get feedback on that.